### PR TITLE
Documented that you can restart CVMs in client and port forwarding

### PR DESCRIPTION
**Pull request by stuclem**
Fixes https://github.com/vmware/vic-product/issues/2186.

@hickeng can you please review?

### CODE REVIEW

**Comment by stuclem** [comment]
Adding @zjs as a reviewer.

**Comment by stuclem** [comment]
This PR also fixes https://github.com/vmware/vic-product/issues/502.

File: docs/user_doc/vic_app_dev/network_use_cases.md [hunk 1]
```diff
@@ -19,12 +20,20 @@ To perform certain networking operations on containers, your Docker environment
...
 
 **Result:**  You can access Port 80 on `test1` from the public network interface on the VCH at port 8080.
 
+## Container VM Port Forwarding <a id="portforwarding"></a>
+
+You can forward a port within a container VM, in the same way you can via NAT on the endpoint VM:
+
+`$ docker run --net=published-container-net -p 80:8080 -d tomcat:alpine`
+
+The above example allows you to access the Tomcat webserver via port 80 on the container VM, via `published-container-net`, instead of being fixed to port 8080 as defined in the Tomcat Dockerfile. This makes it significantly simpler for you to expose services directly via container networks, without having to modify images.
```
Comment (hickeng) [inline docs/user_doc/vic_app_dev/network_use_cases.md]: We need to explain which network the port mapping applies to when a container is connected to both bridge and container networks. The port mapping applies to the network used on the `create` command. It does not apply to networks connected after that.

Actual example mapping in the container VM and serving to the container network:
```
$ docker create -p 80:8080 --net=public tomcat:alpine
50ca681655326775da27462ee062f3a74c4157ff04470fcfda9b176470270d6a
$ docker network connect bridge 50
$ docker start 50
50
$ docker ps
CONTAINER ID        IMAGE               COMMAND             CREATED             STATUS              PORTS                         NAMES
50ca68165532        tomcat:alpine       "catalina.sh run"   7 minutes ago       Up 5 minutes        192.168.78.159:80->8080/tcp   priceless_tesla
```

A note should also be added to the section `Deploy Containers That Combine Bridge Networks with a Container Network`

Comment (stuclem) [reply docs/user_doc/vic_app_dev/network_use_cases.md]: Fixed

**Comment by stuclem** [comment]
Thanks @zjs and @hickeng

**Pull request merged**